README.md

```diff
@@ -1,5 +1,62 @@
 # BC Parks Foundation
 
+BC Parks Foundation is a Mobile App build with React Native Technology that can be deployed in both ios and Android (with some ajustments) platforms.
+
+## <img src="./client/js/assets/images/discoverWhite.png" >
```

Comment:
This image is not loading properly. Please check.

Reply:
Is loading for me